### ISSUE TITLE
Stopped working on iframe about 1 week ago.

### ISSUE THREAD

**Issue by Transisto** [mempool/mempool#1020]
Stopped working on iframe about 1 week ago.

We display mempool.space using Yodeck digital signage system.

this is an error I get. 
`Refused to frame 'https://mempool.space/' because an ancestor violates the following Content Security Policy directive: "frame-ancestors 'none'".`

**Comment by wiz** [comment]
Hello,

We've always had a HTTP Content Security Policy on mempool.space which disallows untrusted frame ancestors (since at least a year ago) - if you can explain more about your use case and exact URL for us to see how it's being framed, we might consider whitelisting it.

**Comment by Transisto** [comment]
Yodeck is a digital signage Os that we run on a rPi.

What I posted comes from the "preview" that can run locally.

I can try pinpointing the exact time this stopped working.

It's used mostly to show customers that there's been no block since X time and teach them how block variability works.

**Comment by wiz** [comment]
Yeah that's cool, but I would need the exact URL to whitelist anything

**Comment by Transisto** [comment]
Hi @wiz, I've revisited the problem, Do you want to know the pages from mempool we're looking to display?

They'd have been the root page,  the /tv , the next blocks at 
"https://mempool.space/mempool-block/0" 
https://mempool.space/mining
https://mempool.space/lightning

**Comment by wiz** [comment]
I'm aware that our URL is mempool.space, I'm asking what frame URLs you need it whitelisted for this RPI platform

**Comment by Transisto** [comment]
I'm having back and forth with support over this, It's hard for me to know since that's all abstracted away as if it's a browser window without an address bar. 

In the meanwhile, I'm curious what's the risk of allowing mempool.space to be rendered in an iframe.
